src/components/Events.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
This is fine now, but this would be a great candidate for react server component since the only state is the date from fetch itself.